.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
-/codeCoverage
\ No newline at end of file
+/codeCoverage
+.idea
```

Comment:
Yes, I've seen it in quite a few places for phpstorm, netbeans etc as the project folders are usually stored within the source tree for portability